src/manager.cpp

```diff
@@ -23,14 +24,51 @@ Manager::Manager(
         // Set up minimal things that is needed before bus name is claimed.
         m_worker->performInitialSetup();
 
-        // set async timer to detect if system VPD is published on D-Bus.
+        // set async timer to detect if system Vpd is published on D-Bus.
```

Comment:
VPD should be all caps in comment. Why this was changed?

Reply:
will modify